packages/paste-core/components/form/src/FormHelpText.tsx

```diff
@@ -3,38 +3,61 @@ import * as PropTypes from 'prop-types';
 import {ValueOf} from '@twilio-paste/types';
 import {Flex} from '@twilio-paste/flex';
 import {Text, safelySpreadTextProps} from '@twilio-paste/text';
+import {TextColor} from '@twilio-paste/style-props';
 import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
 
 export const FormHelpTextVariants = {
   ERROR: 'error',
+  ERROR_INVERSE: 'error_inverse',
+  INVERSE: 'inverse',
+  UNDEFINED: undefined,
 } as const;
 
 export type FormHelpTextVariants = ValueOf<typeof FormHelpTextVariants>;
 
 export interface FormHelperTextProps extends React.HTMLAttributes<HTMLDivElement> {
   className?: never;
+  disabled?: boolean;
   marginTop?: 'space0';
   style?: never;
   variant?: FormHelpTextVariants;
 }
 
-const FormHelpText: React.FC<FormHelperTextProps> = ({marginTop, children, variant, ...props}) => {
+const FormHelpText: React.FC<FormHelperTextProps> = ({marginTop, disabled, children, variant, ...props}) => {
   let icon = null;
   switch (variant) {
     case FormHelpTextVariants.ERROR:
       icon = <ErrorIcon color="colorTextError" decorative size="sizeIcon20" />;
       break;
+    case FormHelpTextVariants.ERROR_INVERSE:
+      icon = <ErrorIcon color="colorTextErrorLight" decorative size="sizeIcon20" />;
+      break;
     default:
       break;
   }
 
+  let textColor = 'colorTextWeak' as TextColor;
+  if (disabled) {
+    textColor = 'colorTextWeaker';
```

Comment:
We can't disable help text I'm afraid, it always has to pass 4.5:1. It can be `weak`, but nothing lighter

Reply:
@SiTaggart does the label on a disabled field need to pass 4.5:1, too? Or just the help text?

Reply:
Both please

Reply:
@SiTaggart So we need drop the style of the disabled labels. Looks like our default labels are also not accessible when disabled then. I can go ahead and make that change in this PR if necessary.

Reply:
Going with `colorTextWeak` and `colorTextInverseWeak`